build/sizes.h

```diff
@@ -23,9 +23,9 @@
 #define SIZES_H
```

Comment:
Is this file used? If so does the double cast function need a declaration?

Reply:
Yes, we use it (though apparently headers are not required for JS use?). It might just be pro forma, but I can add the declaration here.